extensions/aproc/proc/ingest/drivers/impl/dimap.py

```diff
@@ -211,26 +233,31 @@ def to_item(self, url: str, assets: list[Asset]) -> Item:
             item.properties.sensor = metadata["DATASET_PRODUCER_NAME"]
         if "MISSION_INDEX" in metadata:
             item.properties.sensor_type = metadata["MISSION_INDEX"]
+
+        if dim_path != self.dim_path:
+            os.remove(dim_path)
```

Comment:
The system should never delete any file. If it is really needed, then this must be documented and we must use the comment "!DELETE!".

https://docs.google.com/document/d/1UTm5JUZ8F9ptlcya9v88hzg-iPWxeuCCziAD_oycloo/edit?tab=t.0